kolibri/core/assets/src/navigation.vue

```diff
@@ -1,16 +1,16 @@
 <template>
 
-  <header id="navigation-module">
+  <header>
     <nav class="titlebar">
```

Comment:
This in fact should not be a `nav` tag, since it doesn't contain navigation items or a list as it's immediate children. It seems likely you could remove this element entirely. A good project for @DXCanas in a future PR.